src/OrbitGl/GlCanvas.h

```diff
@@ -46,22 +46,15 @@ class GlCanvas : public orbit_gl::AccessibleInterfaceProvider, protected QOpenGL
   virtual void MouseWheelMoved(int x, int y, int delta, bool ctrl);
   virtual void MouseWheelMovedHorizontally(int /*x*/, int /*y*/, int /*delta*/, bool /*ctrl*/) {}
   virtual void RightDown(int x, int y);
-  virtual bool RightUp();
+  virtual void RightUp();
```

Comment:
The returned boolean indicated whether a context menu was supposed to be shown or not. Hence this is not needed anymore.